src/main/resources/org/jenkins/plugins/lockableresources/actions/LockableResourcesRootAction/tableQueue/table.jelly

```diff
@@ -179,6 +191,23 @@ THE SOFTWARE.
                 ${%ago(h.getTimeSpanString(queuedItem.getQueuedTimestamp()))}
               </j:if>
             </td>
+            <td>
+              ${queuedItem.priority}
+            </td>
+            <td>
+              <l:hasPermission permission="${it.UNLOCK}">
+                <button
+                  onClick="changeQueueOrder(this, '${queuedItem.id}');"
+                  class="jenkins-button jenkins-!-success-color"
+                  tooltip="${%btn.changeOrder.detail} ${queuedItem.id}"
+                >
+                  ${%btn.changeOrder}
+                </button>
+              </l:hasPermission>
```

Comment:
Somehow make it clear that "index" == "position".
Either rename the "Index" column to "position", or make sure people understand that changing the "position" is effectively the same as changing the "index".

Because I guarantee that people will confuse the "position" with the "Queue priority" which is _right next to it_.

This can also be resolved by making "Action" column the _second_ column (right after the "index"), or the very _last_ column. As long as it's _not_ right next to "Queue priority".